articles/my-account-and-profile/sso.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Single Sign-On (SSO)"
+description: "Learn how to enable SSO for your SparkPost account and get answers to common SSO questions."
+---
+
+SparkPost supports single sign-on (SSO) through any SAML provider. SSO is enabled via the UI by an Admin level user.
+
+## Enable SSO
+
+Enabling SSO is a 3-step process:
+
+### 1. Enable for your company:
+1. Click on your user name in the upper right corner and select “Account Settings”.
+1. In the “Single Sign-On” section, to the right of “SAML”, click “Provision SAML”. Upload the IdP metadata XML file for your SAML provider and then click the “Provision SAML” button.
+1. To the right of “Single Sign-On”, click “Enable SSO”.
+
+### 2. Enable for each user:
+1. If you have just enabled SSO for your company and are still on “Account Settings” page, click on “Manage single sign-on users” to go to the “Manage Users” page. If you are coming from any other page, click on your user name in the upper right corner and select “Manage Users” to access that page.
+1. Find the user for whom you would like to enable SSO. Click on the three dots in the rightmost column for more options. Click “Edit”.
+1. Check the box next to “Enable single sign-on authentication for this user” and then click “Update User”.
+1. Repeat steps “b” and “c” for each additional user.
+
+### 3. Configure your Identity Provider (IdP) with the callback URL:
+1. Your IdP will need the callback URL for SparkPost. Please refer to your IdP for specific instructions on where to submit this information.
+1. The callback URL is:
+  
+For US-hosted customers: `https://api.sparkpost.com/api/v1/users/saml/consume`
+
+For EU-hosted customers: `https://api.eu.sparkpost.com/api/v1/users/saml/consume`
+
+For Enterprise customers: `https://<host>/api/v1/users/saml/consume`  
+*(Enterprise customers: note that you must replace < host > with your SparkPost tenant name. Please see your TAM if you need assistance determining your callback URL.)*
+
+## FAQ
+
+### What do I need to provide to my IdP?
+The specifics for configuring SSO in your IdP varies by provider, so we recommend you reference their documentation for full details. In general, you will need to provide the callback URL as described above. You may also be asked for metadata in some cases. If so, here is the metadata you can provide to your IdP:
+
+Metadata for US-hosted Customers:
+
+```xml
+<?xml version="1.0"?>
+<EntityDescriptor xmlns="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig# " entityID="passport-saml" ID="passport_saml">
```

Comment:
Looks like the US and EU metadata are the same.

Reply:
good catch! Fixed!